src/test/scala/firrtlTests/ReplSeqMemTests.scala

```diff
@@ -693,4 +693,37 @@ circuit Top :
          |""".stripMargin
     compileAndEmit(CircuitState(parse(input), ChirrtlForm))
   }
+
+  "ReplSeqMem" should "rename blackbox instances to suggested names" in {
+    val input =
+      """
+        |circuit CustomMemory :
+        |  module CustomMemory :
+        |    input clock : Clock
+        |    input reset : UInt<1>
+        |    output io : {flip rClk : Clock, flip rAddr : UInt<3>, dO : UInt<16>, flip wClk : Clock, flip wAddr : UInt<3>, flip wEn : UInt<1>, flip dI : UInt<16>}
+        |    io is invalid
+        |    smem mem_0 : UInt<16>[7]
+        |    read mport _T_17 = mem_0[io.rAddr], clock
+        |    io.dO <= _T_17
+        |    when io.wEn :
+        |      write mport _T_18 = mem_0[io.wAddr], clock
+        |      _T_18 <= io.dI
+        |""".stripMargin
+    val mems = Set(
+      MemConf("renamed_mem", 7, 16, Map(WritePort -> 1, ReadPort -> 1), None)
+    )
+    val confLoc = "ReplSeqMemTests.confTEMP"
+    val annos = Seq(
+      ReplSeqMemAnnotation.parse("-c:CustomMemory:-o:" + confLoc),
+      SuggestSeqMemNameAnnotation(
+        CircuitTarget("CustomMemory")
+          .module("CustomMemory")
+          .ref("mem_0"),
+        "renamed_mem"
+      )
+    )
+    val res = compileAndEmit(CircuitState(parse(input), ChirrtlForm, annos))
+    checkMemConf(res, mems)
+  }
```

Comment:
This test is showing the memory name in `MemConf`, but it would be really helpful if it could also show what the instance and module names are for the wrapper module, as well as the instance of the blackbox.

Reply:
is the goal to change the instance name? I dont think in general we *want* to, but what it was doing before that the instance name was the same as the blackbox name... which IMHO is strange but that's the existing behavior

Reply:
I would say we do **not** want to change the instance name

Reply:
I guess by "change" there is "same as it was before this PR" or "change based on what the person passes in"... which do you mean we do NOT want to do